apps/segment/graphql/subscriptions/order-confirmed.graphql

```diff
@@ -0,0 +1,12 @@
+fragment OrderConfirmedSubscriptionPayload on OrderConfirmed {
+  issuedAt
```

Comment:
as written before I think we should map order creation date

Reply:
I'm wondering what will user expect in Segment (as this will be mapped to `timestamp` property indicating when event happened in time) and I think it will be the time of when `order confirmed` or other event has happened instead of when order was created 🤔

Reply:
As we discussed internally we will use `issuedAt` for the moment as it shouldn't be difference between it and `order update date` (for this case).